packages/next/types/react-dom.d.ts

```diff
@@ -42,7 +42,7 @@ declare module 'react-dom/server.edge' {
     bootstrapModules?: Array<string | BootstrapScriptDescriptor>
     progressiveChunkSize?: number
     signal?: AbortSignal
-    onError?: (error: unknown, errorInfo: unknown) => string | undefined
+    onError?: (error: unknown) => string | undefined
```

Comment:
this is an old type. the linked file is to an old commit. Not sure how we're using this file here but we should probably be tracking the commit of the files we've vendored.